### PR TITLE
YEP-89-Add-proximity-sorting-to-profile-search-routes

### PR DESCRIPTION
**Pull request by Xerataw**

### CODE REVIEW

File: back/src/composables/useUtils.ts [hunk 1]
```diff
@@ -130,6 +147,7 @@ const useUtils = () => ({
   sendResponse,
   toDbFormat,
   fromDbFormat,
+  calculate_distance,
```
Comment (alexis-moins) [inline back/src/composables/useUtils.ts:150]: snake_case whereas other functions are written in camelCase ?

File: back/src/middlewares/authenticate.ts [hunk 2]
```diff
@@ -31,6 +33,8 @@ const authenticate = async (
   req.account = {
     id: account.id,
     profileType: account.profile_type,
+    longitude: profile?.longitude,
```
Comment (alexis-moins) [inline back/src/middlewares/authenticate.ts:36]: Maybe add `|| null` to prevent `undefined` values ?

File: back/src/composables/useUtils.ts (hunk 1 above)
Comment (Xerataw) [reply back/src/composables/useUtils.ts]: It's modified !

File: back/src/middlewares/authenticate.ts (hunk 2 above)
Comment (Xerataw) [reply back/src/middlewares/authenticate.ts]: Yeah I modified it too

**Review by alexis-moins — APPROVED** [review]

**Comment by alexis-moins** [comment]
Good

**Pull request merged**